components/Container/Container.styles.ts

```diff
@@ -7,4 +7,5 @@ export const widths = {
 export const bgColors = {
   black: 'bg-gc-black text-white',
   white: 'bg-white text-gc-black',
+  'fog-light': 'bg-fog-light text-gc-black',
```

Comment:
Add fog light/black text as an option for Container background/text color pairing.